### DISCUSSION TITLE
How stop drag when empty space shows?

### DISCUSSION THREAD

**Discussion by Nazeh-Taha** [xyflow/xyflow#1290] [Q&A] [answered]
How stop drag when empty space shows?

![InkedAnnotation 2021-06-17 145920_LI](https://user-images.githubusercontent.com/54739625/122393326-6d818b00-cf7d-11eb-94ef-6c8c312bd98c.jpg)

**Comment by moklick** [comment, marked as answer]
There is a `translateExtent` prop that defines min and max x and y of the pane https://reactflow.dev/docs/api/component-props/ Is this what you are looking for?

**Reply by Nazeh-Taha** [reply]
Thanks @moklick , this is work 👍
